src/listbox/index.ts

```diff
@@ -177,6 +174,28 @@ export class ListboxBase<P extends ListboxProperties = ListboxProperties> extend
 		}
 	}
 
+	protected animateScroll(scrollValue: number) {
+		this.meta(ScrollMeta).scroll('root', scrollValue);
+	}
+
+	@diffProperty('activeIndex', auto)
+	protected calculateScroll(previousProperties: ListboxProperties, newProperties: ListboxProperties) {
+		this._calculateScroll(newProperties);
+	}
+
+	@afterRender()
+	protected afterRender(dNode: DNode) {
```

Comment:
Could this be done via deferred properties instead?

Reply:
I'm not sure I know what you mean by a deferred property, but it certainly sounds useful in this situation.

Reply:
Addressed this by using `onAttach` instead of an `afterRender`, as this scrolling only needs to be a one time thing